sdk/codes.go

```diff
@@ -0,0 +1,13 @@
+package sdk
+
+type Code uint32
+
+const (
```

Comment:
https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto#L314

why not match with otel?

Reply:
What exactly do you mean? Values and names match, also this is copied from codes.Codes from otel-go.

Reply:
In the proto definition `STATUS_CODE_OK  = 1;`

Reply:
I based on https://github.com/open-telemetry/opentelemetry-go/blob/main/codes/codes.go#L23

Reply:
Changed.

Reply:
I'd say we do what otel go does and not the proto, since that's what the otel agent is based on.